package.json

```diff
@@ -34,6 +34,7 @@
     "iced-runtime": "^1.0.2",
     "more-entropy": ">=0.0.7",
     "progress": "~1.1.2",
+    "sha3": "^2.0.0",
```

Comment:
is this needed?

Reply:
JS doesn't have a native sha3 implementation. I chose this package because it only depends on buffer and it's published by a real company (https://www.phusion.nl/).  Although it shouldn't be a `^`, I'll remove that.

Reply:
where is it used?

Reply:
https://www.npmjs.com/package/js-sha3 has more dependents and no dependencies but it's published by some random person (tm). What do you suggest?